### PR TITLE
fix disappearing sidebar button

### LINKED ISSUE #5564: The "Hide right side bar" button disappears when using the zoom of the browser
## Context
This bug is present in view mode and tracing mode.

## Steps to Reproduce the bug
1. Open any dataset
2. Use the zoom of the browser (Firefox or Chrome)
3. The "Hide right side bar"-button in the upper right corner disappears

## Your Environment for bug
(Linux Chrome and Firefox) and (Mac Firefox)

### PR DESCRIPTION
**Pull request by hotzenklotz**
PR fix a long-standing issues where the little button to toggle the right-hand side panel would disappear when using the browser's builtin zoom. The issue is related to a float vs int comparison. 

### URL of deployed dev instance (used for testing):
- https://sidebarbutton.webknossos.xyz

### Steps to test:
- Load any dataset
- Use the browser zoom
- Before: the little button disappears in some zoom scales and re-appears in others
- Now: little button always here

### Issues:
- fixes #5564

------
(Please delete unneeded items, merge only when none are left open)
- [ ] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] Updated [(unreleased) migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes
- [ ] Adapted [wk-libs python client](https://github.com/scalableminds/webknossos-libs/tree/master/webknossos/webknossos/client) if relevant API parts change
- [ ] Needs datastore update after deployment
- [x] Ready for review

### CODE REVIEW

File: frontend/javascripts/oxalis/view/layouting/flex_layout_helper.ts [hunk 1]
```diff
@@ -60,7 +60,8 @@ export function getPositionStatusOf(tabSetNode: TabSetNode): NodePositionStatus
   // of the root container that contains everything except for the borders.
   isLeftMost = tabSetNodeRect.x === rootContainerRect.x;
   isRightMost =
-    tabSetNodeRect.x + tabSetNodeRect.width === rootContainerRect.x + rootContainerRect.width;
+    tabSetNodeRect.x + tabSetNodeRect.width ===
+    Math.trunc(rootContainerRect.x + rootContainerRect.width);
```
Comment (hotzenklotz) [inline frontend/javascripts/oxalis/view/layouting/flex_layout_helper.ts:64]: rootContainer.x and .width contain fractions in some case. This caused the comparisons to be, .e.g `25 == 25.234543`

Comment (daniel-wer) [reply frontend/javascripts/oxalis/view/layouting/flex_layout_helper.ts:64]: Would it be even more fail-safe to also truncate the `tabSetNodeRect` values in line 63? From the documentation I cannot see why `rootContainerRect` could sometimes be float but `tabSetNodeRect` could not.

**Review by daniel-wer — APPROVED** [review]
Nice find and fix! Please see my one suggestion :)

File: frontend/javascripts/oxalis/view/layouting/flex_layout_helper.ts (hunk 1 above)
Comment (hotzenklotz) [reply frontend/javascripts/oxalis/view/layouting/flex_layout_helper.ts:64]: I have no idea how the fractions are derived. In my testing it worked with 80% - 250% browser scaling without issues.

**Pull request merged**